libbpfgo/libbpfgo.go

```diff
@@ -32,6 +34,15 @@ void set_print_fn() {
 	libbpf_set_print(libbpf_print_fn);
 }
 
+struct ring_buffer * init_ring_buf(int map_fd, int page_cnt) {
+	struct ring_buffer *rb = NULL;
+	rb = ring_buffer__new(map_fd, ringbufferCallback, NULL, NULL); //
+	if (!rb) {
+		fprintf(stderr, "Failed to initialize ring buffer\n");
+        return NULL;
```

Comment:
fixing whitespace right now

Reply:
This brings up a good point though - how should we format the C code? We talked about Go code formatting but I'm curious to hear what you all are using.

Reply:
☝️

Reply:
I still see whitespace issues in the commented C

Reply:
So strange, not sure why that kept happening...